ivy.el

```diff
@@ -3227,12 +3227,20 @@ Otherwise, ~/ will move home."
 
 (defvar ivy--exhibit-timer nil)
 
+(let (last-pos)
+  (defun ivy-input-changed? ()
```

Comment:
The Emacs convention is to end predicate names with `-p`; see [`(info "(elisp) Coding Conventions")`](https://www.gnu.org/software/emacs/manual/html_node/elisp/Coding-Conventions.html).

Reply:
hmm, yh, personally i do prefer `?` from scheme, but okay will change!